node/src/chain_spec.rs

```diff
@@ -294,3 +294,137 @@ fn testnet_genesis(
 		valve: ValveConfig { start_with_valve_closed: false },
 	}
 }
+
+pub fn neumann_latest() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Neumann Network",
+		// ID
+		"neumann",
+		ChainType::Live,
+		move || {
+			let allocation_json = &include_bytes!("../../distribution/neumann_alloc.json")[..];
```

Comment:
does this get prioritized over the 3 big accounts? ie will each of the 3 accounts below (yours, mine and chris') have < 333.33MM NEU?

Reply:
I am going to take the output from subquery and transform it into the allocation json. Our accounts will be in there with whatever amount we have now. We only use this json for endowed accounts